spinoso-env/README.md

```diff
@@ -80,7 +80,7 @@ This crate requires [`std`], the Rust Standard Library.
 All features are enabled by default:
 
 - **system-env** - Enable an `ENV` backend that accesses the host system's
-  environment variables via the [`std::env`](module@std::env) module.
```

Comment:
this looks like a copy-paste error out of the `lib.rs` documentation.

This syntax is used to differentiate the `std::env` module from the `env!` macro.